src/main/java/frc/robot/utils/Tracer.java

```diff
@@ -0,0 +1,211 @@
+package frc.robot.utils;
```

Comment:
Where did we get this?

Reply:
this is just the folder it's in

Reply:
Yeah, I know how Java packages work 😬

I'm asking if we got this code from somewhere else. Was this from another team?

Reply:
ohhh sorry yeah this is from 3173, should i credit them?